EXTCAP.md

```diff
@@ -0,0 +1,180 @@
+```
+pip install pyspinel
+```
+#### Manual installation(for pyspinel 1.0.0a3 and before)
+**1. Install pyspinel package**
+``` 
+pip install pyspinel 
+```
+**2. Install extcap script on Wireshark**
+
+To find the correct installation path of the extcap utility on any system please open Wireshark:
+```
+"Help" -> "About Wireshark" -> "Folders" -> "Extcap path"
+```
+Copy the provided ```extcap_ot.py```, ```extcap_ot.bat``` and ```sniffer.py``` to the extcap directory.
+
+For OS X and Linux, verify that the ```extcap_ot.py``` file has the “x” permission. If not, add it using:
+
+```
+chmod +x extcap_ot.py
+```
+
+For Windows, verify that the ```extcap_ot.bat``` file is in the extcap folder
+
+## Usage
+
+### NAME
+    extcap_ot.py - external script for integrating OpenThread Sniffer with Wireshark
+
+### SYNOPSIS
+    extcap_ot.py [--arguments]
+
+### DESCRIPTION
+
+```
+    -h, --help            
+    	Show this help message and exit
+
+    --extcap-interfaces
+       	Provide a list of interfaces to capture from.
+
+    --extcap-interface <EXTCAP_INTERFACE>
+        Provide the interface to capture from.
+    
+    --extcap-dlts
+        Provide a list of dlts for the given interface
+
+    --extcap-config
+        Provide a list of configurations for the given interface.
+
+    --fifo <FIFO>
+        Use together with capture to provide the fifo to dump data to.
+    
+    --channel <CHANNEL>
+         IEEE 802.15.4 capture channel [11-26].
+
+    --baudrate <BAUDRATE>
+        Set the serial port baud rate.
+
+    --tap
+        Use to specify DLTs as IEEE 802.15.4 TAP (only for Wireshark3.0 and later).
+```
+
+## Quick Start
+### Configuring Wireshark for Thread
+* Wireshark configuration - [Protocols](https://openthread.io/guides/ncp/sniffer#wireshark_configuration_-_protocols)
+* Wireshark configuration - [FCS Format](https://openthread.io/guides/ncp/sniffer#wireshark_configuration_-_rssi)
+    *  For Wireshark 2.x: TI CC24xx metadata
+    * For Wireshark 3.x: ITU-T CRC-16
+   
+### Using the Sniffer
+#### Wireshark capture screen
+The Wireshark capture screen is displayed when Wireshark is first launched. 
+It includes the Wireshark interface for managing packets that are captured 
+and the hardware interfaces connected to the OT Sniffer.
+#### Start sniffing
+
+There are three ways to start sniffing:
+* If this is your first time using an interface, click on interface options 
+  to set channel and baudrate, then click start. Check the IEEE 802.15.4 tap 
+  to ensure that the channel information is included in the pcap output and 
+  can be displayed in the Wireshark GUI. The parameters will be saved after 
+  the start of the capture, and you will not need to set it again the next 
+  time you use the interface (unless you need to change the channel)
+
+* Double click on the hardware interface
+* Select the hardware interface by clicking on it and then click the Wireshark 
```

Comment:
No need to say "by clicking on it", just "Select the hardware interface and then click the Wireshark..."